dashboard/app/controllers/puzzle_ratings_controller.rb

```diff
@@ -0,0 +1,15 @@
+class PuzzleRatingsController < ApplicationController
+  def create
+    status_code = :unauthorized
```

Comment:
minor: I'd prefer an early return in the not enabled? case so that a reader doesn't have to go through the whole method (even though it is only 3 actual lines long ;)) to see where the :unauthorized goes